lib/jekyll-remote-theme/theme.rb

```diff
@@ -91,9 +121,8 @@ def gemspec
 
       def valid_hosts
         @valid_hosts ||= [
-          "github.com",
-          ENV["PAGES_GITHUB_HOSTNAME"],
-          ENV["GITHUB_HOSTNAME"],
+          host,
+          default_host,
```

Comment:
As currently written, this is an anti-abuse mechanism, which it looks like this change removes. In an untrusted environment, a malicious actor could request a remote theme from a malicious Git server, leading to a denial of service or other attack. How would we limit hosts in untrusted environments?

Reply:
The problem for us is that since we are using custom DNS, privately hosted repos, the themes are not hosted on any of theses endpoints.